css-two-column-layout/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row content-center">
+      <h1>SCALE, SPEED, AGILITY</h1>
+    </div>
+    <div class="row">
+      <div class="intro-container">
+        <p class="text-center line-height-20">We've dedicated thousands of man hours to making healthcare institutions in the Helium network
+          run 10x more efficiently and deliver an unparalleled quality of care.</p>
+      </div>
+    </div>
+    <div class="row wrap margin-top">
+      <div class="column-half">
+        <div class="column-third">
+          <div class="icon-container">
+            <img class="icon" src="images/software.svg">
+          </div>
+        </div>
+        <div class="column-two-third text-container-top">
+          <div class="description">
+            <p class="about-title">
+              INTUITIVE, </br>
+              SMART SOFTWARE
+            </p>
+            <p class="line-height-20">
+              We build products that transform how healthcare facilities run across Africa - making them radically more efficient operationally and financially, a complete turnaround.
+            </p>
+          </div>
+        </div>
```

Comment:
You don't want to have `column` elements immediately inside other `column` elements. You should make a `row` class element to hold these `column` elements. So like this:

```
row
  column-half
    row
      column-third
      column-two-third
```